ext/hal/nordic/nrfx/soc/nrfx_coredep.h

```diff
@@ -32,6 +32,8 @@
 #ifndef NRFX_COREDEP_H__
 #define NRFX_COREDEP_H__
 
+#include <drivers/nrfx_common.h>
```

Comment:
I am affraid it is not allowed to modify nrfx like that. It is an external library so you would need to start PR there:
https://github.com/NordicSemiconductor/nrfx
and wait until it will be in Zephyr.

Reply:
Hmm - will try to resolve it somehow.